sys/fuchsia/guest.txt

```diff
@@ -12,5 +12,8 @@ resource zx_guest[zx_handle]
 # continues to mean nothing.
 zx_guest_create(handle zx_resource, options int32, guest ptr[out, zx_guest], vmar ptr[out, zx_vmar])
 
-# TODO: teach this how to use ports & keys
 zx_guest_set_trap(handle zx_guest, kind int32, vaddr intptr, size int64, port_handle zx_port, key int64)
+# TODO: page-align `addr and `size` for BELL and MEM.
```

Comment:
It looks relatively easy to do, in https://github.com/google/syzkaller/blob/master/docs/syscall_descriptions_syntax.md: `int64[0:18446744073709551615, 4096]`, I think.

Reply:
Thank you. Added via 2 types.